docs/modules/ROOT/pages/documentation/payara-server/logging/logging.adoc

```diff
@@ -32,3 +32,17 @@ image:logging/logging_setup.png[Logger Settings]
 
 xref:/documentation/payara-server/logging/daily-log-rotation.adoc[Daily log rotation] +
 Documentation about daily log rotation in Payara and how to configure it.
+
+[[env-var-replacement]]
+== ENV variable replacement
```

Comment:
Can we also change it here?
```suggestion
== Environment variable replacement
```

Otherwise all OK, I approve.

Reply:
Thx, missed that (it was late in the evening :) )